logstash/class-logger.php

```diff
@@ -516,7 +516,7 @@ public static function maybe_wp_debug_log_entries( array $entry ) : void {
 	 * @param array $entry Data.
 	 */
 	public static function wp_debug_log( array $entry ) : void {
-		if ( ! defined( 'VIP_GO_ENV' ) || ! VIP_GO_ENV ) {
+		if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
The check didn't match the description ("Don't run this on VIP Go"). The intention is to avoid VIP Go sites to attempt to write to `wp-content/debug.log`